fix/fixer.go

```diff
@@ -48,6 +48,7 @@ func init() {
 		"galaxy-command":             new(FixerGalaxyCommand),
 		"comm-config":                new(FixerCommConfig),
 		"ssh-wait-timeout":           new(FixerSSHTimout),
+		"vsphere-iso-net-disk":       new(FixerVSphereNetworkDisk),
```

Comment:
Nit: split into separate fixers: `vsphere-iso-nic` and `vsphere-iso-storage`

While the code is nearly identical, the fields are different enough to keep them as separate fixers. From a maintainers' and users' perspective it think it would be a little easier to grok what each fixer is changing.

Reply:
I had thought about it initially, but since it is doing the exact same thing at once, I didn't think it warranted a separate fixer. If the changes were coming out at different times, it would make more sense.

Reply:
Sound good to me :+1: